server_tests/test/clientTests.js

```diff
@@ -715,7 +715,11 @@ describe("RUST API TEST SUITE", function() {
                 fileUri = res.body.results.fileUri;
                 return testClient.getImage(fileUri);
             })
-            .then((res) => validate.done(res.body, schema.names.commandGetImage))
+            .then((res) => {require('istextorbinary').isBinary(null, res.body, function(err, res) {
```

Comment:
If this is commonly used (seems to be needed at least twice) it may be a good idea to throw it into util.js as a helper function.